docs/application-development-guide/src/main/asciidoc/securing-apps.adoc

```diff
@@ -1545,9 +1545,9 @@ or in a directory of your choice. For easy reference throughout the rest
 of this section, this directory is referred to as simply app-dir.
```

Comment:
It seems this part is completely obsoleted. Maybe we should rewrite it and bind it to the https://github.com/eclipse-ee4j/jakartaee-examples repository. There are no samples in the distribution any more.